scripts/tcl_commands/all.tcl

```diff
@@ -223,10 +227,15 @@ proc trim_lib {args} {
     set_if_unset arg_values(-input) $::env(LIB_SYNTH_COMPLETE)
     set_if_unset arg_values(-output) $::env(LIB_SYNTH)
 
+    set no_synth_list "$::env(NO_SYNTH_CELL_LIST)"
+    if { [info exists $::env(NO_SYNTH_CELL_LIST)] } {
```

Comment:
is this `file exists` instead of `info exists`

Reply:
something else is not right here. i am not sure what

Reply:
Yeah, the variable is wrong. oops.